helm/templates/deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    service: {{ .Release.Name }}
+    app: service
+spec:
+  replicas: {{ .Values.replicas }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: {{ .Values.rollingUpdate.maxSurge }}
+      maxUnavailable: {{ .Values.rollingUpdate.maxUnavailable }}
+  minReadySeconds: {{ .Values.minReadySeconds }}
+  revisionHistoryLimit: {{ .Values.revisionHistoryLimit }}
+  template:
+    metadata:
+      labels:
+        service: {{ .Release.Name }}
+        app: service
+    spec:
+      containers:
+      - name: service
+        image: gaiapipeline/gaia:{{ .Values.imageTag }}
+        imagePullPolicy: {{ .Values.imagePullPolicy }}
+        lifecycle:
+          # @see https://github.com/kubernetes/contrib/issues/1140
+          preStop:
+            exec:
+              command: ["sleep", "{{ .Values.preStop.sleep }}"]
+        env:
+        - name: GAIA_PORT
+          value: "8080"
+        - name: GAIA_HOSTNAME
+          value: http:0.0.0.0
```

Comment:
shouldn't this be `https://0.0.0.0`?

Reply:
I think the better way is to use `https` only in ingresses if needed. You should also keep in mind that for current kubernetis custom certs should be used.

Reply:
What I meant is that the `//` is missing from here.